appinventor/appengine/build.xml

```diff
@@ -623,6 +625,29 @@
     <mkdir dir="${build.extra.dir}" />
     <java failonerror="true" fork="true" classname="com.google.gwt.dev.Compiler">
       <classpath>
+        <pathelement location="src"/>
+        <pathelement location="${build.dir}/components/ComponentTranslation/src"/>
+        <pathelement location="${build.war.dir}/WEB-INF/classes"/>
+        <pathelement location="${local.build.dir}/AiRebindLib.jar" />
+        <pathelement location="${build.dir}/common/CommonUtils-gwt.jar" />
+        <pathelement location="${build.dir}/common/CommonVersion-gwt.jar" />
+        <pathelement location="${build.dir}/components/CommonConstants-gwt.jar" />
+        <!-- external libs -->
+        <pathelement location="${lib.dir}/findbugs/jsr305.jar" />
+        <pathelement location="${lib.dir}/guava/guava-20.0.jar" />
+        <pathelement location="${lib.dir}/guava/guava-gwt-20.0.jar" />
+        <pathelement location="${lib.dir}/guava/error_prone_annotations-2.0.12.jar" />
+        <pathelement location="${lib.dir}/guava/j2objc-annotations-1.1.jar" />
+        <pathelement location="${lib.dir}/gwt_dragdrop/gwt-dnd-3.2.3.jar" />
+        <pathelement location="${lib.dir}/gwt_incubator/gwt-incubator-20101117-r1766.jar" />
+        <pathelement location="${lib.dir}/gwt_query/gwtquery-1.5-beta1.jar" />
+        <pathelement location="${lib.dir}/gwt_svg/lib-gwt-svg-0.5.15.jar" />
+        <pathelement location="${lib.dir}/charba/charba-2.5.jar" />
+        <!-- gwt libs -->
+        <pathelement location="${gwt.sdk}/gwt-dev.jar"/>
+        <pathelement location="${gwt.sdk}/gwt-user.jar"/>
+        <pathelement location="${gwt.sdk}/validation-api-1.0.0.GA.jar" />
+        <pathelement location="${gwt.sdk}/validation-api-1.0.0.GA-sources.jar" />
```

Comment:
Compare and contrast these entries with the entries in `YaClientApp.classpath` above, as it is referenced using path refid on the following line. All classpath items for YaClientApp should be defined in the above entry instead of here (this classpath is duplicated multiple places in the file so this centralizes the reference).